vertx-web-client/src/main/asciidoc/index.adoc

```diff
@@ -210,6 +210,28 @@ You can also write headers using putHeader
 {@link examples.WebClientExamples#sendHeaders2(io.vertx.ext.web.client.WebClient)}
 ----
 
+=== Configure the request to add authentication.
```

Comment:
add a note that user should use this with HTTPS

Reply:
added text: 

> Authentication can be performed manually by setting the correct headers, or, using our predefined methods
(We strongly suggest having HTTPS enabled, especially for authenticated requests):